lambda_backend/route_microservice/src/common/schema.ts

```diff
@@ -12,7 +12,7 @@ export const createRouteSchema = {
       properties: {
         countryCode: {
           type: 'string',
-          maxLength: 50,
+          maxLength: 3,
           pattern: AlphanumericSpaceHyphen,
```

Comment:
If we're using ISO country code won't the pattern just be alpha/alphanumeric?

Reply:
yea but i think its okay LOL

Reply:
fixed